### PR TITLE
Enable hot module reloading for webpack reporter

### PR DESCRIPTION
**Pull request by georgeweiler**
This update enables hot module reloading for the electrode-webpack-reporter when changes to the app are saved.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "electrode-webpack-reporter",
-  "version": "0.2.4",
+  "name": "gweiler-electrode-webpack-reporter",
```
Comment (ananavati) [inline package.json]: this should be `electrode-webpack-reporter`

**Review by ananavati — CHANGES_REQUESTED** [review]
* lint fixes and the package name change.
* related pr in [generator-electrode](https://github.com/electrode-io/electrode/tree/master/packages/generator-electrode)

**Comment by georgeweiler** [comment]
related PR: https://github.com/electrode-io/electrode/pull/66

File: lib/webpack-reporter.js [hunk 2]
```diff
@@ -18,6 +19,7 @@ function removeCwd(s) {
 }
 
 const reporterStats = require("../dist/server/stats.json");
+let socketIO;
```
Comment (jchip) [inline lib/webpack-reporter.js]: this should be an instance field of WebpackReporter.

**Review by ananavati — APPROVED** [review]

**Pull request merged**